apps/src/applab/ApplabVisualizationColumn.jsx

```diff
@@ -0,0 +1,55 @@
+var Visualization = require('./Visualization');
+var GameButtons = require('./GameButtons');
+
+var styles = {
+  hidden: {
+    display: 'none'
+  }
+};
+
+/**
+ * Equivalent of visualizationColumn.html.ejs. Initially only supporting
+ * portions used by App Lab
+ */
+var VisualizationColumn = React.createClass({
+  propTypes: {
+    isProjectLevel: React.PropTypes.bool.isRequired,
+    isSubmittable: React.PropTypes.bool.isRequired,
+    isSubmitted: React.PropTypes.bool.isRequired,
+  },
+
+  render: function () {
+    return (
+      <div>
+        <Visualization/>
+        <GameButtons
+            isProjectLevel={this.props.isProjectLevel}
+            isSubmittable={this.props.isSubmittable}
+            isSubmitted={this.props.isSubmitted}
+        />
+        <div id="belowVisualization">
```

Comment:
I would turn this into a `ProtectedStatefulDiv` now, since it is/contains elements that StudioApp may bind handlers against, and you tore out the wrapping PSD above.  You can just give it an id instead of a content function.